eng/targetingpacks.targets

```diff
@@ -185,4 +160,6 @@
                         Condition="'%(RuntimeFramework.FrameworkName)' == '$(LocalFrameworkOverrideName)'" />
     </ItemGroup>
   </Target>
+
+  <Import Project="$(MSBuildThisFileDirectory)readytorun.targets" Condition="'$(UseLocalTargetingRuntimePack)' == 'true'" />
```

Comment:
The targetingpacks.targets file is designed to be self-contained. Can we just inline the target and make `Microsoft.NETCore.App.Runtime.sfxproj` import targetingpacks.targets? Alternatively, if we don't want that, can we move this import out of this file, similar to how we use the live linker (just another import next to targetingpacks.targets)?

Reply:
We can’t as we need to override the target, not sequence before or after it. I can move it out, but I wanted to have it trigger on the same properties that are set in this file so that if you’re using the live runtime, you get the live R2R experience as well.